domapping/cli.py

```diff
@@ -50,7 +50,8 @@ def cli():
               help='Mapping generation configuration.')
 @click.option('--indent', '-i', default=4, type=click.INT,
               help='Output json indentation step.')
-def schema_to_mapping_cli(schema, output, config, indent):
+@click.option('--final', '-f', default=False, is_flag=True)
```

Comment:
I wonder if we could find a better name than `final`. It let the user think that without this the mapping is not finished, even though it can be used without any issue.